### PR TITLE
World generation mappings

### PR DESCRIPTION
**Pull request by mschae23**
World generation mappings for 1.19, mostly for `ChunkNoiseSampler`,  `BlendingData` and `Blender`.

### CODE REVIEW

File: mappings/net/minecraft/world/gen/chunk/Blender.mapping [hunk 1]
```diff
@@ -53,9 +61,10 @@ CLASS net/minecraft/class_6748 net/minecraft/world/gen/chunk/Blender
 	METHOD method_40027 (IILorg/apache/commons/lang3/mutable/MutableDouble;Lorg/apache/commons/lang3/mutable/MutableDouble;Lorg/apache/commons/lang3/mutable/MutableDouble;Ljava/lang/Long;Lnet/minecraft/class_6749;)V
 		ARG 5 chunkPos
 		ARG 6 data
-	CLASS class_6781
-		METHOD get (Lnet/minecraft/class_6749;III)D
+	CLASS class_6781 BlendingSampler
+		METHOD get sample (Lnet/minecraft/class_6749;III)D
```
Comment (apple502j) [inline mappings/net/minecraft/world/gen/chunk/Blender.mapping]: Please do not rename those.

Comment (mschae23) [reply mappings/net/minecraft/world/gen/chunk/Blender.mapping]: Is that for technical reasons or a convention?

Comment (apple502j) [reply mappings/net/minecraft/world/gen/chunk/Blender.mapping]: Both, I think. They aren't assigned intermediary names, so things might break.

Comment (mschae23) [reply mappings/net/minecraft/world/gen/chunk/Blender.mapping]: Alright, I'll remove them.

File: mappings/net/minecraft/world/gen/chunk/BlendingData.mapping [hunk 2]
```diff
@@ -1,22 +1,135 @@
...
 	FIELD field_35519 SURFACE_BLOCKS Ljava/util/List;
 	FIELD field_35682 CODEC Lcom/mojang/serialization/Codec;
-	FIELD field_35692 heights [D
-	METHOD method_39347 (ILnet/minecraft/class_2791;II)V
+	FIELD field_35683 BIOMES_PER_CHUNK I
+	FIELD field_35684 LAST_CHUNK_BIOME_INDEX I
+	FIELD field_35685 CHUNK_BIOME_END_INDEX I
+	FIELD field_35686 NORTH_WEST_END_INDEX I
+	FIELD field_35687 SOUTH_EAST_END_INDEX_PART I
+	FIELD field_35690 initialisedBlendingData Z
```
Comment (apple502j) [inline mappings/net/minecraft/world/gen/chunk/BlendingData.mapping]:
```suggestion
	FIELD field_35690 initializedBlendingData Z
```

**Review by apple502j — CHANGES_REQUESTED** [review]

**Review by apple502j — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**